addons/stock/static/src/js/inventory_adjustments_list_controller.js

```diff
@@ -0,0 +1,36 @@
+/** @odoo-module alias=stock.InventoryAdjustmentsListController**/
+
+import SingletonListController from 'stock.SingletonListController';
+
+const InventoryAdjustmentsListController = SingletonListController.extend({
+    buttons_template: 'StockInventoryAdjustments.Buttons',
+
+    // -------------------------------------------------------------------------
+    // Public
+    // -------------------------------------------------------------------------
+
+    init: function (parent, model, renderer, params) {
+        this.context = renderer.state.getContext();
+        return this._super(...arguments);
+    },
+
+    /**
+     * @override
+     */
+    renderButtons: function ($node) {
+        this._super(...arguments);
+        this.$buttons.on('click', '.o_button_apply_all', this._onApplyAll.bind(this));
+    },
+
+    _onApplyAll: function () {
+        this.do_action('stock.action_stock_inventory_adjustement_name', {
+            additional_context: {
+                'active_model': 'ir.ui.view',
+                'active_ids': this.initialState.data.map(el => el['res_id']),
+            }
+        });
+    },
+});
+
+export default InventoryAdjustmentsListController;
+
```

Comment:
Ca fait beaucoup pour une feature pas super interéssante. Check si on peut pas annuler ce point

Reply:
@amoyaux  Sinon, peut-être aller placer ça directement dans le `SingletonListController` ?
Quitte à renommer `singleton_list` (et tout ce qui s'y rapporte) en `inventory_adjustments_list` car `inventory_adjustments_list` semble plus pertinent.

Je pense pas qu'il y ait d'endroit où on veut explicitement le comportement du `SingletonList` sans cet ajout.
Du coup, ça n'impliquerait que l'ajout des 3 méthodes dans un fichier qui existe déjà, au lieu de créer deux nouveaux fichiers JS.

Reply:
Ouep pourquoi pas. Après ça me choque pas de devoir sélectionner les lignes a valider surtout que le filtre 'To Apply' existe

Reply:
I agree with Steve actually. I think the button still adds some value, the select all + apply selected seems a bit unintuitive to me and could actually be used quite a lot.

Reply:
Ok